deploying/metadatastore.rst

```diff
@@ -0,0 +1,47 @@
+Deploying metadatastore
+-----------------------
+
+Lessons learned from SRX and CSX
+================================
+
+Beamlines will have /DATA and /POOL data storage. /DATA seems to be for
+experimental data and /POOL for analysis.  SRX and CSX have opted to run the
+mongo database daemon on the channel archiver server (xf05id-ca1, xf23id-ca).
+This seems to be a reasonable setup and will be repeated on other beamlines
+until someone provides a better option.  We found it useful to set up an alias
+as CSX called 'xf23id-broker' that metadatastore and filestore could point to
+so that they dont actually care which machine the mongodb daemon is running on.
+We should do the same for SRX and all future beamlines, unless a better option
+is presented.  Contact Petkus to do this.
+
+Proposed Deployment Strategy
+============================
+
+#. Set up an alias at beamline for the server running the mongo daemon
+   (xf##id-broker)
+#. Install and configuration of mongodb on xf##id-broker
+
+    #. Install mongodb on xf##id-broker::
+
+        sudo apt-get install mongodb
+
+    #. Edit /etc/mongodb.conf so that ``dbpath=/DATA/mongodb/`` and
+       ``bind_ip = 0.0.0.0``
```

Comment:
There are tons of options that we could use for the configuration file. Maybe, we should go over these together and write up a small draft so that the counsel would discuss...